docs: note commit messages don't support scope (#16435)
eslint/eslint: docs/src/developer-guide/contributing/pull-requests.md

git commit
```

> All ESLint projects follow [Conventional Commits](https://www.conventionalcommits.org/) (**except**: the optional scope is not supported) for our commit messages. Here's an example commit message:

Reviewer: To fit better with our overall docs style:

```suggestion
All ESLint projects follow [Conventional Commits](https://www.conventionalcommits.org/) for our commit messages. (Note: we don’t support the optional scope in messages.)  Here's an example commit message:
```
